filters/builtin/creationmetrics.go

```diff
@@ -31,10 +31,14 @@ func NewRouteCreationMetrics(metrics filters.Metrics) *RouteCreationMetrics {
 
 // Do implements routing.PostProcessor and records the filter creation time.
 func (m *RouteCreationMetrics) Do(routes []*routing.Route) []*routing.Route {
+	return m.reportRouteCreationTimes(routes, time.Now())
+}
+
+func (m *RouteCreationMetrics) reportRouteCreationTimes(routes []*routing.Route, now time.Time) []*routing.Route {
 	for _, r := range routes {
 		for origin, start := range m.startTimes(r) {
 			if m.initialized {
-				m.metrics.MeasureSince(metricsPrefix+origin, start)
+				m.metrics.MeasureDuration(metricsPrefix+origin, now.Sub(start)/100)
```

Comment:
I find a factor of `100` here very confusing. If we do any factor, it should be `1000`. However, I do not see that we gain a real value by this change. It make the values in the buckets intransparent and it actually violates the contract of the measurement bucket, which is to receive values in seconds.

Reply:
I agree that it might be confusing, maybe 1000 is better than 100.
It's a workaround to shift to buckets, which are more fine grained.
If we want to drop the workaround we would need to increase the buckets (IIRC the step is 60s,120s, so 80s would be in one of these buckets and not counted individually), which costs, memory. Skipper memory in our setup is fine, but Prometheus is already suffering.
Maybe you can enhance the apiMonitoring metrics in general to have them in a separate keyspace.